pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -539,6 +539,7 @@ func (c *Controller) checkNodeTopologyDistribution() {
 	nodes, err := c.nodeLister.List(labels.Everything())
 	if err != nil {
 		klog.Errorf("Error listing Nodes: %v", err)
+		return
```

Comment:
@robscott I think we should return on error here, doesn't make sense of syncing after an error on the informer, right?
_unrelated to the PR, but since I found it here_ :)

Reply:
Yeah, that's bizarre, not sure why I'm not returning an error there, but we should be.